modules/ftp-client/src/main/java/org/dcache/ftp/client/dc/AbstractDataChannel.java

```diff
@@ -122,7 +122,7 @@ public static void registerHandler(int transferMode,
 	}
 	*/
 
-        logger.debug("registering handler for class " + clazz.toString() + "; id = " + id);
+        logger.debug("registering handler for class {}; id = {}", clazz.toString(), id);
```

Comment:
"toString" on any object is called implicitly whenever you log or print to console. No need for explicit toString call